worlds/ladx/__init__.py

```diff
@@ -514,7 +514,13 @@ def remove(self, state, item: Item) -> bool:
         return change
 
     def get_filler_item_name(self) -> str:
+        if self.options.stabilize_item_pool:
         return "Nothing"
+        else:
+            # Same fill choices and weights used in LADXR.itempool.__randomizeRupees
+            filler_choices = ('Bomb', 'Single Arrow', '10 Arrows', 'Magic Powder', 'Medicine')
+            filler_weights = ( 10,     5,              10,          10,             1)

```

Comment:
Also, instead of recreating these tuples every time you call `get_filler_item_name`, you can just store these as a constant somewhere and use those